.github/workflows/device-tests-android.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           path: bin
 
       - name: Setup Gradle
-        uses: gradle/actions/setup-gradle@ed408507eac070d1f99cc633dbcf757c94c7933a # v4.4.3
+        uses: gradle/actions/setup-gradle@4d9f0ba0025fe599b4ebab900eb7f3a1d93ef4c2 # v5.0.0
```

Comment:
note: breaking change is
> Upgrade to node 24

see https://github.com/gradle/actions/releases/tag/v5.0.0